.github/workflows/setup-homebrew.yml

```diff
@@ -35,18 +35,24 @@ jobs:
     steps:
       - name: Checkout
         run: |
-          # Allow writing under linuxbrew user
-          orig_uid=$(ls -nd $GITHUB_WORKSPACE | awk 'NR==1 {print $3}')
-          sudo chown -R $(whoami) $GITHUB_WORKSPACE $HOME
+          if [ "$RUNNER_OS" = "Linux" ]
+          then
+            # Allow writing under linuxbrew user
+            orig_uid="$(stat --printf='%u' "$GITHUB_WORKSPACE")"
+            sudo chown -R "$(whoami)" "$GITHUB_WORKSPACE" "$HOME"
+          fi
```

Comment:
CC @Bo98 -- just want to make sure gating this behind Linux makes sense.

Reply:
Not Bo, but I would say yes.

Reply:
Yep, that's fine.